validphys2/src/validphys/utils.py

```diff
@@ -5,12 +5,42 @@
 @author: Zahari Kassabov
 """
 import contextlib
+import functools
 import pathlib
 import shutil
 import tempfile
+import typing
+from typing import Any, Sequence, Mapping
```

Comment:
Hate to be difficult, but please make this consistent. I.e. just `import typing` or import `Hashable` to the local namespace.

Reply:
Black and isort still have to be run on these changes

Reply:
I should have been clearer, what I meant was being consistent with the importing of Any, Sequence, Mapping, and Hashable, which is not something that isort will change. So doing the following
```suggestion
from typing import Any, Sequence, Mapping, Hashable
```